PostHog/PostHogSDK.swift

```diff
@@ -468,6 +472,9 @@ let maxRetryDelay = 30.0
 
             queue.add(event)
 
+            // Automatically set person properties for feature flags during identify event
+            setPersonPropertiesForFlagsIfNeeded(userProperties)
```

Comment:
we should `sanitizeDictionary(userProperties)` here too since we do that already, we can cache in a var and reuse the result here